lib/phoenix/digester.ex

```diff
@@ -46,18 +47,48 @@ defmodule Phoenix.Digester do
     |> Enum.map(&(map_file(&1, input_path)))
   end
 
-  defp generate_manifest(files, output_path) do
+  defp load_digests(input_path) do
+    manifest_path = Path.join(input_path, "manifest.json")
```

Comment:
This is correct, however in the case that we are building a digest from an old manifest file we still need the `input_path` to read existing digested files.